src/main/java/uk/gov/pay/api/model/directdebit/DirectDebitConnectorCreatePaymentRequest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.pay.api.model.directdebit;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class DirectDebitConnectorCreatePaymentRequest {
+    public final int amount;
+    public final String reference;
+    public final String description;
+    public final String mandateId;
+
+    private DirectDebitConnectorCreatePaymentRequest(DirectDebitConnectorCreatePaymentRequestBuilder builder) {
+        this.amount = builder.amount;
+        this.reference = builder.reference;
+        this.description = builder.description;
+        this.mandateId = builder.mandateId;
+    }
+
+    @JsonProperty("amount")
+    public int getAmount() {
+        return amount;
+    }
+
+    @JsonProperty("reference")
+    public String getReference() {
+        return reference;
+    }
+
+    @JsonProperty("description")
+    public String getDescription() {
+        return description;
+    }
+
+    @JsonProperty("agreement_id")
```

Comment:
This has to stay as "agreement_id" for now as it's what dd-connector expects. We can update later